packages/editor/src/store/actions.js

```diff
@@ -177,7 +177,13 @@ export function* setupEditor( post, edits, template ) {
 	};
 	yield resetEditorBlocks( blocks, { __unstableShouldCreateUndoLevel: false } );
 	yield setupEditorState( post );
-	if ( edits ) {
+	if (
+		edits &&
+		Object.keys( edits ).some(
+			( key ) =>
+				edits[ key ] !== ( has( post, [ key, 'raw' ] ) ? post[ key ].raw : post[ key ] )
```

Comment:
The `raw` key seems very API specific. Given all the API abstractions, is this the right place to check for the `raw` key?

Reply:
Yeah, it's not ideal. I think we should deprecate initial edits if possible.